2017-11-23 00:24:45,519: log.pubsub.bridge.0000.child: log.pubsub.bridge.0000.child    : MainThread     : INFO    : python.interpreter   version: 2.7.13 (default, Jan 19 2017, 14:48:08) [GCC 6.3.0 20170118]
2017-11-23 00:24:45,519: log.pubsub.bridge.0000.child: log.pubsub.bridge.0000.child    : MainThread     : INFO    :                      pid: 7790
2017-11-23 00:24:45,519: log.pubsub.bridge.0000.child: log.pubsub.bridge.0000.child    : MainThread     : INFO    :                      tid: MainThread
2017-11-23 00:24:45,519: log.pubsub.bridge.0000.child: log.pubsub.bridge.0000.child    : MainThread     : INFO    : radical.pilot        version: 0.46.2
2017-11-23 00:24:45,519: log.pubsub.bridge.0000.child: log.pubsub.bridge.0000.child    : MainThread     : INFO    : start bridge log.pubsub.bridge.0000.child on tcp://*:*
2017-11-23 00:24:45,521: log.pubsub.bridge.0000.child: log.pubsub.bridge.0000.child    : MainThread     : INFO    : bound bridge log.pubsub.bridge.0000.child to tcp://0.0.0.0:45359 : tcp://0.0.0.0:44467
2017-11-23 00:45:41,806: log.pubsub.bridge.0000.child: log.pubsub.bridge.0000.child    : MainThread     : WARNING : send failed ([Errno 9] Bad file descriptor) - terminate
